`make upload-assets` no longer works

```
$ make upload-assets
venv/bin/fluffy-upload-assets
make: venv/bin/fluffy-upload-assets: Command not found
make: *** [Makefile:30: upload-assets] Error 127
```

This broke in 9e91c1ca0a72f7adc6fd04980084dd765e4f5d8d when we stopped installing the dev version of the app (so that requirements-tools would work properly).

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ export FLUFFY_SETTINGS := $(CURDIR)/settings.py
 minimal: $(VENV) assets settings.py
 
 $(VENV): setup.py cli/setup.py requirements.txt requirements-dev.txt
-	vendor/venv-update venv= -p python3 venv install= -r requirements.txt -r requirements-dev.txt -e cli
+	vendor/venv-update venv= -p python3 venv install= -r requirements.txt -r requirements-dev.txt -e cli -e .
 
 fluffy/static/app.css: $(VENV) $(wildcard fluffy/static/scss/*.scss)
 	$(BIN)/sassc fluffy/static/scss/app.scss $@

